src/main/java/au/org/ala/biocache/dao/SolrIndexDAOImpl.java

```diff
@@ -84,6 +80,19 @@
   @Inject
   private FieldMappingUtil fieldMappingUtil;
 
+  /*
+  * The csv header for updating solr
+  */
+  private final List<String> header = Arrays.asList(
```

Comment:
the SOLR index field should be `userAssertions`. We've adopted camel case for all fields in the search index.